conf/locale/es_419/LC_MESSAGES/djangojs.po

```diff
@@ -9787,10 +9829,13 @@ msgstr ""
 
 #: openedx/features/learner_profile/static/learner_profile/templates/share_modal.underscore
 msgid ""
-"{download_link_start}Download this image (right-click or option-click, save "
-"as){link_end} and then {upload_link_start}upload{link_end} it to your "
-"backpack."
+"%(download_link_start)sDownload this image (right-click or option-click, "
+"save as)%(link_end)s and then %(upload_link_start)supload%(link_end)s it to "
+"your backpack.</li>"
```

Comment:
Is this closing tag a part of string? Please verify this.

Reply:
Fixed! Thanks!